### DISCUSSION TITLE
Which parameter to set to simulate a constant wave disturbance?

### DISCUSSION THREAD

**Discussion by Londu** [OpenFAST/openfast#1187] [Q&A]
Hello,

I need to design a DAC controller for an offshore 5MW reference wind turbine. The wind and wave disturbances need to be accommodated in the design. 
For the linearization, the WaveMod in Hydrodyn has to be set to 0 (still water). 
My question is, which parameter needs to be set if I want to simulate constant wave disturbances? 
Thank you in advance. 

Regards

**Comment by jjonkman** [comment]
Hi @Londu,

The wave disturbance input to the linearized system derived from OpenFAST is expressed as the wave elevation from still water level.  To include this wave disturbance, the state-space-based wave-excitation model from the potential-flow solution must be enabled (`PotMod` = 1, `ExctnMod` = 2), with a suitable state-space based wave-excitation model provided (specified via `PotFile`).  The following paper should clarify the approach: https://www.nrel.gov/docs/fy19osti/71865.pdf.

Best regards,

**Reply by Londu** [reply]
Hi, Mr. jjonkman,

Thank you very much for your reply. My model is for an offshore fixed-bottom wind turbine. The PotMod and ExctnMod are under the floating platform section in HydroDyn.  Will it still give the expected disturbances? 

Best regards

**Reply by jjonkman** [reply]
Hi @Londu,

How are you modeling the hydrodynamics of your fixed-bottom wind turbine?  Via the potential-flow solution or via the strip-theory (Morison) solution?  (If you are modeling a fixed-bottom monopile, I suspect you are using the strip-theory solution.)  At this time, the wave disturbance input to the linearized system derived from OpenFAST is only valid for the potential flow solution.

Best regards,
